src/kusto/azext_kusto/generated/_help.py

```diff
@@ -1,758 +1,798 @@
+--principal-assignment-name "kustoprincipal1" --resource-group "kustorptest" --created
+      - name: Pause executing next line of CLI script until the kusto cluster-principal-assignment is successfully \
+updated.
+        text: |-
+               az kusto cluster-principal-assignment wait --cluster-name "kustoclusterrptest4" \
+--principal-assignment-name "kustoprincipal1" --resource-group "kustorptest" --updated
+      - name: Pause executing next line of CLI script until the kusto cluster-principal-assignment is successfully \
+deleted.
+        text: |-
+               az kusto cluster-principal-assignment wait --cluster-name "kustoclusterrptest4" \
+--principal-assignment-name "kustoprincipal1" --resource-group "kustorptest" --deleted
+"""
+
+helps['kusto database'] = """
+    type: group
+    short-summary: kusto database
+"""
+
+helps['kusto database list'] = """
+    type: command
+    short-summary: "Returns the list of databases of the given Kusto cluster."
+    examples:
+      - name: KustoDatabasesListByCluster
+        text: |-
+               az kusto database list --cluster-name "kustoclusterrptest4" --resource-group "kustorptest"
+"""
+
+helps['kusto database show'] = """
+    type: command
+    short-summary: "Returns a database."
+    examples:
+      - name: KustoDatabasesGet
+        text: |-
+               az kusto database show --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--resource-group "kustorptest"
+"""
+
+helps['kusto database create'] = """
+    type: command
+    short-summary: "Creates or updates a database."
+    parameters:
+      - name: --read-write-database
+        short-summary: "Class representing a read write database."
+        long-summary: |
+            Usage: --read-write-database soft-delete-period=XX hot-cache-period=XX location=XX kind=XX
+
+            soft-delete-period: The time the data should be kept before it stops being accessible to queries in \
+TimeSpan.
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+      - name: --read-only-following-database
+        short-summary: "Class representing a read only following database."
+        long-summary: |
+            Usage: --read-only-following-database hot-cache-period=XX location=XX kind=XX
+
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+    examples:
+      - name: KustoDatabasesCreateOrUpdate
+        text: |-
+               az kusto database create --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--parameters "{\\"location\\":\\"westus\\",\\"properties\\":{\\"softDeletePeriod\\":\\"P1D\\"}}" --resource-group \
```

Comment:
@astauben --parameters is an unrecognized argument.

Reply:
fixed. Thanks!